### ISSUE TITLE
remove fire animation from stealth

### ISSUE THREAD

**Issue by CelestialKnight7** [cs6413110/Pixel-Tanks#17]
do

**Comment by cs6413110** [comment]
nah, take a vote

**Comment by CelestialKnight7** [comment]
k

**Comment by CelestialKnight7** [comment]
@BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam @BreadSpam 
@BreadSpam 
VOTE DEMOCRACY NEEDS YOU

**Comment by CelestialKnight7** [comment]
iF YOU VOTE INCORRECTLY I WILL NERF BUIDER

1 turret
less damage per bullet
aim deviation
12 ammo
automatically dies after 20 seconds

**Comment by LostKing73** [comment]
bruh

**Comment by LostKing73** [comment]
uhm, like why does it need to be harder to see stelths.

**Comment by CelestialKnight7** [comment]
cuz stealth was balanced around not being able to see them while fire animation was broken and now that's its back it has an advantage* over stealth

*I know ur gonna say "WeLl DuH" but I mean its an unfair advantage as now its put stealth in a worse place as (previously stated) stealth was balanced around not being able to see them.

Furthermore no other class has a class that explicitly counters them, I mean you could argue that warrior counters tactical with its i-frames but even then that's not a direct counter and is something the tactical can work around. Stealth meanwhile, if set on fire is basically ruined until the fire goes out or they die. This also makes it extremely annoying to fight against which is not great for game design. In any other class you could fight any other class and have a good time, but in this interaction between stealth and fire the entire game practically gets ruined. It almost feels like an abusive relationship with fire just bulling stealth every time they're in the same lobby. fire still somewhat has an advantage over stealth anyways as stealth will boost away when at low hp but the fire (most of the time) will kill them after they escape (in my experience). 

The main way to counter a class that we seem to be using is items, airstrike counters builder, reflect - tactical, block - warrior (arguable but shut), dynamite - stealth (again arguable but you get my point). Having items counter classes has also been balanced, if as a tactical you encounter someone with reflect, while it does counter you when used well - its ultimately up to the tactical to play around the reflect and wait for them to waste it before firing the insta. having a class that counters another class makes playing against fire as a stealth almost impossible. From the constant uptime with burning combined with the fact that half of your class (the invisibility) is ruined, there is no clever way to play around it similar to tactical with reflect.

This system of having a class counter another class is also annoying for non-stealths as well. Imagine you're playing builder and a stealth starts harassing you, you might think to switch to fire to deal with them but this shouldn't be how the game is designed. If you had to switch your preferred playstyle to be efficient it would force everyone to leave and rejoin everytime another class appears. This goes the same way for stealth, lets say you harass a builder and then they switch to fire, you would want to switch your class so that you wouldn't be harassed by the fire. This system creates not only a toxic environment as everyone would be switching their class so that they don't get countered or to counter someone else, it would also not be fun as people wound't be able to play the class they want.

In conclusion, if the fire animation was invisible while the stealth was invisible the entire class wouldn't be countered by fire and would help to create a game where any strategy could be used and be efficient. 

Thank you for coming to my Ted talk.

**Comment by cs6413110** [comment]
Ok, but only if stealth gets a nerf where it can't infinitely flat.

**Comment by DarkMemeGod411** [comment]
so in my opinion, fire ruins stelth

**Comment by CelestialKnight7** [comment]
> Ok, but only if stealth gets a nerf where it can't infinitely flat.

invisibility from fire and being able to go through walls are almost 2 different topics altogether, you could increase the minimum requirement for the to boost from 1 energy (idk why u set it to this) to 5 energy (the orignial amount) furthermore you could make it so that stealthboosting could cost 5 energy regardless and if you don't have the required energy you just do a normal boost( I think this is how it is rn already with the normal boost)

**Comment by LostKing73** [comment]
eh aaron and evan figure it out idc

**Comment by BreadSpam** [comment]
ok so what about stealth countering builder???

**Comment by LostKing73** [comment]
lol skill issue bread

**Comment by BreadSpam** [comment]
K voted bye

**Comment by cs6413110** [comment]
SAVE

**Comment by LostKing73** [comment]
why reopen? its done

**Comment by BreadSpam** [comment]
is it?

**Comment by DarkMemeGod411** [comment]
ye its done

**Comment by CelestialKnight7** [comment]
yayyyy
